docs/design/design-patterns/distributed-system-design-reference/README.md

```diff
@@ -0,0 +1,9 @@
+# Distributed System Design Reference
+
+Distributed systems introduce new and interesting problems that need addressing.
+Software engineering as a field has dealt with this problem for years, and there are phenomenal resources available for reference when creating a new distributed system.
```

Comment:
```suggestion
Software engineering as a field have dealt with these problems for years, and there are phenomenal resources available for reference when creating a new distributed system.
```

Reply:
Thanks for your review. I think that `has` is correct here, because `Software engineering` is singular.

Reply:
```suggestion
Software engineering as a field has dealt with these problem for years, and there are phenomenal resources available for reference when creating a new distributed system.
```

Reply:
I went a little overboard with the have :) you're right, software engineering is singular... was mainly trying to fix the these (referring to the problems - which are multiple based on the sentence above)